Doc/library/calendar.rst

```diff
@@ -422,7 +430,7 @@ For simple text calendars this module provides the following functions.
 .. function:: calendar(year, w=2, l=1, c=6, m=3)
 
    Returns a 3-column calendar for an entire year as a multi-line string using
-   the :meth:`formatyear` of the :class:`TextCalendar` class.
+   :meth:`TextCalendar.formatyear`.
```

Comment:
Alternative:

```suggestion
   the :meth:`~TextCalendar.formatyear` of the :class:`TextCalendar` class.
```

Reply:
Simplification abandoned; restored previous text and adjusted the link in 277460eb6fff051d8ad7b681c6abf5d027fa6db8.